app/services/AuthService.ts

```diff
@@ -13,5 +17,27 @@ export const AuthService = {
 			return true;
 		}
 		return false;
+	},
+
+    async validateLogin(uid: string, password) {
+
+		
+	    var data
+		await axios.post('hslinux:38383/api/v1/auth', {
+		uid: uid,
+		password: password
+		  })
+		  .then(function (response) {
+		data = console.log(response.data);
+			
```

Comment:
Try returning the axios value rather than storing it in data. 

In the future, make sure to initialize the values as well. 

In general, don't have console.logs you used for debugging.